test-utils/src/commonMain/kotlin/blank.kt

```diff
@@ -1,3 +1,4 @@
+@file:Suppress("ktlint:standard:no-empty-file")
```

Comment:
If the problem in the folder to keep, can't we just create empty file `.keep` in this folder?

Reply:
Very quick test, we cannot:

> Execution failed for task ':test-utils:generateMetadataFileForIosSimulatorArm64Publication'.
> java.io.FileNotFoundException: /Users/gijsvanveen/projects/kaluga/test-utils/build/classes/kotlin/iosSimulatorArm64/main/klib/test-utils.klib (No such file or directory)